### DISCUSSION TITLE
Error in Wazuh on login

### DISCUSSION THREAD

**Discussion by SejalSheth** [wazuh/wazuh#22715] [General]
This morning when i logged into Wazuh, I am getting the below error. Please advise how do I troubleshoot the below error. 

An internal server error occurred.
Version: 2.8.0
Build: 47202
Error: Internal Server Error
    at Fetch._callee3$ (https://xxxxx/47202/bundles/core/core.entry.js:15:585158)
    at tryCatch (https://xxxxx/47202/bundles/plugin/customImportMapDashboards/customImportMapDashboards.plugin.js:13:786910)
    at Generator.invoke [as _invoke] (https://xxxxx/47202/bundles/plugin/customImportMapDashboards/customImportMapDashboards.plugin.js:13:790926)
    at Generator.next (https://xxxxx/47202/bundles/plugin/customImportMapDashboards/customImportMapDashboards.plugin.js:13:788105)
    at fetch_asyncGeneratorStep (https://xxxxx/47202/bundles/core/core.entry.js:15:578070)
    at _next (https://xxxxx/47202/bundles/core/core.entry.js:15:578386)

**Comment by Ipfi90** [comment]
Hi,

could you fix this problem?
We're having the same issue since end of the last week.

**Comment by SejalSheth** [comment]
Unfortunately not, I am still looking for resolution.

Thanks
Sent from my iPhone

On Apr 15, 2024, at 03:17, Ipfi90 ***@***.***> wrote:

﻿
You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

Hi,

could you fix this problem?
We're having the same issue since end of the last week.

**Comment by Ipfi90** [comment]
Hi, just to let you know, I was able to fix the problem on my side.

On the Wazuh Dashboard Server I get the following events, when I restart the wazuh-dashboard service with systemctl restart wazuh-dashboard.

WZ3 opensearch-dashboards[12843]: {"type":"log","@timestamp":"2024-04-15T11:43:57Z","tags":["error","opensearch","data"],"pid":12843,"message":"[circuit_breaking_exception]: [parent] Data too large, data for [<http_reque>
Apr 15 11:43:58 WZ3 opensearch-dashboards[12843]: {"type":"log","@timestamp":"2024-04-15T11:43:58Z","tags":["error","opensearch","data"],"pid":12843,"message":"[circuit_breaking_exception]: [parent] Data too large, data for [<http_reque>
Apr 15 11:43:58 WZ3 opensearch-dashboards[12843]: {"type":"log","@timestamp":"2024-04-15T11:43:58Z","tags":["error","savedobjects-service"],"pid":12843,"message":"Unable to retrieve version information from OpenSearch nodes."}
Apr 15 11:44:03 WZ3 opensearch-dashboards[12843]: {"type":"log","@timestamp":"2024-04-15T11:44:03Z","tags":["error","opensearch","data"],"pid":12843,"message":"[circuit_breaking_exception]: [parent] Data too large, data for [<http_reque>
Apr 15 11:44:03 WZ3 opensearch-dashboards[12843]: {"type":"log","@timestamp":"2024-04-15T11:44:03Z","tags":["error","savedobjects-service"],"pid":12843,"message":"Unable to retrieve version information from OpenSearch nodes."}
Apr 15 11:44:05 WZ3 opensearch-dashboards[12843]: {"type":"log","@timestamp":"2024-04-15T11:44:05Z","tags":["error","opensearch","data"],"pid":12843,"message":"[circuit_breaking_exception]: [parent] Data too large, data for [<http_reque>
Apr 15 11:44:08 WZ3 opensearch-dashboards[12843]: {"type":"log","@timestamp":"2024-04-15T11:44:08Z","tags":["error","opensearch","data"],"pid":12843,"message":"[circuit_breaking_exception]: [parent] Data too large, data for [<http_reque>
Apr 15 11:44:10 WZ3 opensearch-dashboards[12843]: {"type":"log","@timestamp":"2024-04-15T11:44:10Z","tags":["error","opensearch","data"],"pid":12843,"message":"[circuit_breaking_exception]: [parent] Data too large, data for [<http_reque>
Apr 15 11:44:13 WZ3 opensearch-dashboards[12843]: {"type":"log","@timestamp":"2024-04-15T11:44:13Z","tags":["error","opensearch","data"],"pid":12843,"message":"[circuit_breaking_exception]: [parent] Data too large, data for [<http_reque>
Apr 15 11:44:15 WZ3 opensearch-dashboards[12843]: {"type":"log","@timestamp":"2024-04-15T11:44:15Z","tags":["error","opensearch","data"],"pid":12843,"message":"[circuit_breaking_exception]: [parent] Data too large, data for [<http_reque>


I've then found this:
https://groups.google.com/g/wazuh/c/prIHE47mxX8

On my side the xms and xmx settings were xms1g and xmx1g - I've increased both to 8 and restarted the indexer / dashboard service

Hope this helps on your side too
